tox.ini

```diff
@@ -51,9 +51,6 @@ commands =
 deps =
     -rrequirements.txt
     -rrequirements-dev.txt
-    coverage
```

Comment:
I'm not sure why this was here instead of in `requirements-dev.txt` where it seems to belong, maybe @john-bodley knows more about this.

Reply:
For some context https://github.com/apache/incubator-superset/pull/4552

Reply:
I think `requirements-dev.txt` has simply grown to contain more than just the base requirements. If coverage will only be run as part of the CI then it makes sense to have it in `tox.ini` otherwise `requirements-dev.txt` makes sense.

Reply:
@john-bodley it sounds like you have no objections, and if that's the case I'll leave it in `requirements-dev.txt`. I like to run unit tests, linter, and codecov quite often in my dev workflow, and its good to have everything in place and low friction for those quick iterative type of flows.